mod/cornercutter/ModFeature/GlobalSetting/BestRunBlocker.cs

```diff
@@ -39,4 +40,20 @@ static RunData CloneRunData(RunData originalRunData)
             return clonedRunData;
         }
     }
+
+    [HarmonyPatch(typeof(GameManager), nameof(GameManager.IsBestTime))]
+    class BestTimeImageBlocker
+    {
+        static bool Prefix(ref bool __result)
+        {
+            CutterConfig cornercutter = CutterConfig.Instance;
+            GlobalOptions globals = cornercutter.GlobalOptions;
+            if (!cornercutter.CornercutterIsEnabled()) return true;
+            if (!globals.HasFlag(GlobalOptions.DisableHighscores)) return true;

```

Comment:
I've noticed in some of the following scripts you combine this into a single if statement, which might be nice for consistency. This also applies to the `BestRunBlocker` above.